app/models/user.rb

```diff
@@ -525,6 +513,21 @@ def hotness_score
     search_score
   end
 
+  def connected_authentication_providers
+    identities_enabled.pluck(:provider).map(&:to_sym)
+  end
+
+  def authenticated_social_accounts
```

Comment:
This only seems to be used in one view so I'm very reluctant to add this to the model. Could it be in a helper/decorator instead?